src/layout.js

```diff
@@ -52,9 +52,10 @@ let Dimensions;
 
 
 /**
- * Set or cached browser natural dimensions for elements. The tagname
- * initialized here will return true `hasNaturalDimensions`, even if yet to be
- * calculated. Exported for testing.
+ * Set of cached browser natural dimensions for elements. The tagname
```

Comment:
Both descriptions do not talk to the API. It should read simply "Returns the natural dimensions of an element with the specified name. This operation can only be completed for an element whitelisted by `hasNaturalDimensions`".

Or something like this. I'd certainly appreciate if you'd update this.